batch/avro-file/src/main/java/AvroSourceSample.java

```diff
@@ -64,7 +64,7 @@ private void setup() {
     private static Pipeline buildPipeline() {
         Pipeline p = Pipeline.create();
 
-        p.drawFrom(AvroSources.files(AvroSinkSample.DIRECTORY_NAME, User.class, true))
+        p.drawFrom(AvroSources.files(AvroSinkSample.DIRECTORY_NAME, User.class))
```

Comment:
By default, `sharedFileSystem == false`. We have two members in 1 jvm, it will be processed twice.

Reply:
fixed